modules/common/src/DatasetInfo.php

```diff
@@ -135,6 +135,34 @@ public function gather(string $uuid) : array {
     return $info;
   }
 
+  /**
+   * Get the current distribution UUID for a dataset.
+   *
+   * @param string $uuid
+   *   The uuid of a dataset.
+   * @param string $index
+   *   The index of the resource in the dataset array. Defaults to first.
+   *
+   * @return string
+   *   The distribution UUID
+   */
+  public function getDistributionUuid(string $uuid, string $index = '0'): string {
+    $metadata = $this->gather($uuid);
```

Comment:
Similarly, rename `$metadata` to `$dataset_info` to match the docs from `gather()`, and the naming used in the test below.